src/architect/project/project.utils.ts

```diff
@@ -0,0 +1,229 @@
+import fs from 'fs-extra';
+import inquirer from 'inquirer';
+import yaml from 'js-yaml';
+import fetch from 'node-fetch';
```

Comment:
why are we using fetch over axios? In general I'd like to be consistent about the lib as opposed to devs have to choose and then learn both.